### ISSUE TITLE
error when browsing to retrieve.php

### ISSUE THREAD

**Issue by FreeFrags** [isuftin/spotweb#1]
Cant seem to get this up and running on my qnap.
i get the following error when browsing to retrieve.php:
SpotWeb v0.68.29.32 on PHP v5.5.12-2ubuntu4.6 crashed Fatal error occured retrieving reports: Permission denied [21] for objectid [] #0 {main}

==> /var/log/apache2/error.log <==
[Mon Oct 19 15:16:31.962545 2015] [:error] [pid 56] [client 10.0.3.1:53534] PHP Notice: Undefined variable: retriever in /var/www/spotweb/retrieve.php on line 321
[Mon Oct 19 15:16:31.962613 2015] [:error] [pid 56] [client 10.0.3.1:53534] PHP Fatal error: Call to a member function quit() on a non-object in /var/www/spotweb/retrieve.php on line 321

i hope you can help me with this thanks in advance

please see the screenshot for the settings i used.
![spotweb settings](https://cloud.githubusercontent.com/assets/9535811/10643561/d9a2759a-7823-11e5-990b-4e4655343db5.jpg)

**Comment by isuftin** [comment]
@FreeFrags I don't believe retrieve.php is meant to be run via the browser. The authors of SpotWeb seem to have made this to be run via commandline.

The Docker container should have retrieve.php running in a crontab once per hour. 

Please let me know if this is not working for you by checking /var/log/spotweb-retrieve.log on the running Spotweb docker container. I can update the Docker image with a better way to set up a crontab.

You can also manually trigger retrieve by using:

```
docker exec spotweb_server /usr/bin/php /var/www/spotweb/retrieve.php
```

(substitute `spotweb_server` with whatever the spotweb server docker container name is on your system).

I've also included in the `scripts/` directory a file named `trigger-retrieve.sh` - This will attempt to automatically call the retrieve.php file on your docker container.

**Comment by FreeFrags** [comment]
You are right, the retrieve should be called like you said thank you very much.

It looks like the cron job doesnt work here though... still a total beginner with docker though so ill find out how to look at the log

**Comment by isuftin** [comment]
@FreeFrags I've updated the Docker image for spotweb_server which properly creates the crontab. Currently, it seems like Ubuntu is [having issues on their end](https://hub.docker.com/r/isuftin/spotweb/builds/bpdqgtha6vhtt38ecfxzcmp/). 

Because of this, the Docker image will not automatically build on Dockerhub. I will try to kickstart a build on Dockerhub later today in hopes that it does properly build. If this continues to be an issue today, I can also try upgrading the Ubuntu version this Docker container is based on.

You're also free to try and build it locally on your system by cloning this repository, going into the spotweb_server directory and running `$ docker build -t isuftin/spotweb .`

If all goes well, you should have a new Docker image named isuftin/spotweb.

**Comment by FreeFrags** [comment]
cool thank you for your help. i will give it a try as soon as i can

**Comment by isuftin** [comment]
@FreeFrags I'll give a shout out once the new image is built if you'd rather wait for that.

**Comment by FreeFrags** [comment]
alright thank you i will wait for that.

**Comment by isuftin** [comment]
@FreeFrags The latest Docker image is now building Ubuntu 15.10. Go ahead and give that a try. If it works for you, please close this issue. Otherwise, let me know if you still have issues with it.

**Comment by FreeFrags** [comment]
It looks like it is still not retrieving automatically. 
Or did i understand this incorrectly? do i need to setup a job to do this on the host?

**Comment by mrginogit** [comment]
Was this ever resolved?  I just installed on Ubuntu 16.04.1 and my server is not retrieving new spots either.
